DFC.Digital/DFC.Digital.Data/Model/SendEmailRequest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace DFC.Digital.Data.Model
+{
+    public class SendEmailRequest
+    {
+        public string TemplateName { get; set; }
+
+        public string Message { get; set; }
+
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public string Email { get; set; }
+
+        public DateTime DateOfBirth { get; set; }
+
+        public string PostCode { get; set; }
+
+        public string ContactAdviserQuestionType { get; set; }
+
+        public string FeedbackQuestionType { get; set; }
+
+        public bool IsContactable { get; set; }
+
+        public string ContactOption { get; set; }
+
+        public bool TermsAndConditions { get; set; }
+    }
+}
```

Comment:
This model seems tightly coupled to contact us, we need to think about using dictionary for properties or have generic `<T>` with tightly coupled model